### PR TITLE
LiveBlog updates

### PR DESCRIPTION
**Pull request by mparent61**
- Add fields: `LiveBlog.pinned_content`, `LiveBlogResponse.internal_name`
- Fix LiveBlog model indexing (via `excludes`)
- Extra tests

### CODE REVIEW

File: example/testcontent/models.py [hunk 1]
```diff
@@ -109,3 +109,8 @@ def get_absolute_url(self):
 
 class TestLiveBlog(Content, AbstractLiveBlog):
     """Example LiveBlog implementation"""
+
+    class Mapping(Content.Mapping):
```
Comment (mparent61) [inline example/testcontent/models.py]: I suspect this mapping needs to be added to each concrete implementation (AVC, ON, etc) b/c DJES doesn't support multiple base class Mappings?

**Comment by mparent61** [comment]
@benghaziboy @MichaelButkovic @kand   ready for review

File: example/testcontent/models.py (hunk 1 above)
Comment (MichaelButkovic) [inline example/testcontent/models.py]: yeah, this will overwrite each base contents mapping (`class Mapping(BaseContent.Mapping):`)

**Comment by MichaelButkovic** [comment]
LGTM

**Pull request merged**